bootstrap/bootstrap/src/bootstrap.rs

```diff
@@ -57,26 +59,15 @@ fn boostrap_zos(cfg: &config::Config) -> Result<()> {
 
     // write down boot info for other system components (like upgraded)
     flist.write(FLIST_INFO_FILE)?;
-    std::fs::write(FLIST_NAME_FILE, format!("{}/{}", FLIST_REPO, flist.name))?;
+    std::fs::write(FLIST_NAME_FILE, format!("{}/{}", ZOS_REPO, flist.name))?;
 
     install_package(&flist)
 }
 
-/// bootstrap stage install and starts all zos daemons
-pub fn bootstrap(cfg: &config::Config) -> Result<()> {
-    debug!("runmode: {:?}", cfg.runmode);
-    let result = WorkDir::run(WORKDIR, || -> Result<()> {
-        boostrap_zos(cfg)?;
-        Ok(())
-    })?;
-
-    result
-}
-
 /// update stage make sure we are running latest
 /// version of bootstrap
 pub fn update(cfg: &config::Config) -> Result<()> {
-    let result = WorkDir::run(WORKDIR, || -> Result<()> {
+    let result = WorkDir::run(WORKDIR, || {
         update_bootstrap(cfg.debug)?;
```

Comment:
debug is always false because `update()` caller will return if debug is true, no?
hence, name will always be "bootstrap:latest.flist", right?
https://github.com/threefoldtech/zos/blob/124e113e40e09c00c9fa9aca9e2fc9816f837810/bootstrap/bootstrap/src/bootstrap.rs#L90-L97

Reply:
Actually very good catch. that's absolutely correct. I think there were code before that made sense of this but is gone now. 

I think the best way is remove the debug flag. in the update function.

Reply:
Anyway, the logic was that if starting bootstrap in debugging avoid updating yourself usually to test local code so it skips this stage. 
Means we never need to use the development build